scripts/release.py

```diff
@@ -268,17 +268,19 @@ def publish_release(self):
             else:
                 raise
 
-    def update_milestones(self):
-        """Create GitHub release page for tag"""
+    def update_milestones(self, next_version=None):
```

Comment:
What do you mean, you think milestones and releases are not separable, and the update_milestones should be done during instantiation? (therefore move to _init_ ) ?